### ISSUE TITLE
Quastions about Loss

### ISSUE THREAD

**Issue by kascesar** [guigzzz/Keras-Yolo-v2#6]
I have seen other loss functions implemented, yours is more similar to that of Yolo V1, my question is:

1) Why not use categorical crosss entrophy in class predictions ?
2) In the IoU filter, maked in the computation of noobjs_loss you put that
`higthst_iou = tf.math.reduce_max(iou, axis=-1)`
`        mask = (tf.cast(higthst_iou < 0.6, dtype=tf.float32)[..., None] * (1 - true_object))  # noobj mask`
but, if we have to penalize higest IoU, why put < 0.6 and not > 0.6

thanks a lot! :D
